mon-pix/app/styles/components/_challenge-actions.scss

```diff
@@ -82,19 +81,14 @@
       margin-bottom: 0;
       margin-right: auto;
     }
-  }
 
-  &__timed-out {
-    font-family: $font-roboto;
-    font-size: 1rem;
-    font-weight: 600;
-    color: $grey-70;
-    letter-spacing: 0.031rem;
-    margin-right: 20px;
+    &--focused-out-of-window {
+      max-width: 600px;
+    }
```

Comment:
Fonctionnellement, on obtient ça sur les petits écrans 
<img width="426" alt="Screenshot 2021-09-01 at 10 53 21" src="https://user-images.githubusercontent.com/46371288/131642959-324e8a54-46de-4679-b825-a4982ed642f6.png">
 
**suggestion :** Fusionner `&--focused-out-of-window` dans `&__alert-message`

Reply:
Le blocage est coté visuel ? Trop collé sur le coté ?

Reply:
Oui blocage visuel, le texte déborde. Merci pour la question

Reply:
Le problème me semble résolu:
<img width="357" alt="Screenshot 2021-09-06 at 12 07 00" src="https://user-images.githubusercontent.com/11294487/132200527-586fae4a-b7aa-40cd-b0cf-678a6d53cb1f.png">